src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -289,17 +290,13 @@ def create_zero_click_payment(self, order, credit_card_token, installment=None):
             }
 
         response_json = self._call_api(url, payload)
-
-        if not response_json:
+        success = response_json.get("status") == "SUCCESS"
+        if not success:
```

Comment:
```suggestion
        if response_json.get("status") != "SUCCESS"
```

You don't reuse the `success` variable after in this method

Reply:
thanks for the trick reminder :ok_hand: